scripts/travis/build.sh

```diff
@@ -0,0 +1,5 @@
+echo "Installing dependencies..."
+pip install pandas
+pip install matplotlib~=3.4.2
```

Comment:
```suggestion
```
If it has a version, don't install it directly. Such dependencies won't be updated by Dependabot and thus tests won't be accurate. If this is due to a dependency conflict with `SciencePlots`, install requirements/requirements.txt and *then* install the dev requirements.